### PR TITLE
Make updates to sync with the new project name

### PR DESCRIPTION
**Pull request by ayushjn20**
Since the repository name has recently been changed, this PR updates the import statements and relevant links to make the develop branch functional again.
TODO:
Check if 
- [ ] Codacy integration badge is still working in README.md.
- [x] Travis builds still get successful.

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -1,332 +0,0 @@
-# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```
Comment (apoorvam) [inline Gopkg.lock]: Why is `Gopkg.lock` file deleted?

Comment (ayushjn20) [reply Gopkg.lock]: Oh, that is a blunder. Apparently, `dep ensure` did not generate `Gopkg.lock` the file automatically if deleted. I was trying to update the dependencies as well as support for Golang v1.12.x. I had to make an empty lock file and then run `dep ensure`.

Comment (ayushjn20) [reply Gopkg.lock]: I have re-commited changes, please check.

Comment (apoorvam) [reply Gopkg.lock]: There are some new dependencies added here, like `protobuf`. Are those required?

**Comment by apoorvam** [comment]
- [ ] check if all package managers still work correctly, with new name

File: Gopkg.lock (hunk 1 above)
Comment (ayushjn20) [reply Gopkg.lock]: Yes. It's a lock file, that means it is automatically generated. I have updated all the dependencies by deleting the lock file first, then initializing an empty one followed bt `dep ensure`

**Comment by ayushjn20** [comment]
>     * [ ]  check if all package managers still work correctly, with new name

**Comment by apoorvam** [comment]
@ayushjn20 Some of it references the name of the project/url. For example, https://github.com/leopardslab/homebrew-dunner/blob/master/Formula/dunner.rb#L14

It could be similar in all package managers. Github does redirect in some cases, but it's better to verify.

**Comment by ayushjn20** [comment]
I guess there is snapcraft package manager is left, apart from Homebrew as of now.
@agentmilindu Could you please check on the snapcraft portal?

**Pull request merged**